packages/api/lambdas/index-from-database.js

```diff
@@ -9,8 +9,7 @@ const { Search } = require('../es/search');
 const indexer = require('../es/indexer');
 
 const getEsRequestConcurrency = (event) => {
-  const concurrency = event.esRequestConcurrency
-    || process.env.ES_CONCURRENCY;
+  const concurrency = event.esRequestConcurrency || process.env.ES_CONCURRENCY;
```

Comment:
Is there any chance that `event.esRequestConcurrency` could be `0`? this would not behave as expected in that case.

Reply:
It's not very clear, but I think I'm intentionally making sure that we fallback to the env var (or ultimately the default value of 10), because the value is used for `p-limit` below and `p-limit` gives this error if you use 0:

```
Uncaught TypeError: Expected `concurrency` to be a number from 1 and up
    at module.exports (/Users/mboyd/development/cumulus/packages/api/node_modules/p-limit/index.js:6:9)
```

Any ideas on how to improve?

Reply:
I added a comment and a test for this behavior: ff9b192d4e50164e4d5d4620cf145a4890177627

Reply:
```js
const getEsRequestConcurrency = (event) => {
  if (!isNil(event.esRequestConcurrency)) {
    const parsedValue = Number.parseInt(event.esRequestConcurrency, 10);

    if (Number.isInteger(parsedValue) && parsedValue > 0) {
      return parsedValue;
    }

    throw new TypeError('event.esRequestConcurrency must be an integer greater than 0');
  }

  if (!isNil(process.env.ES_CONCURRENCY)) {
    const parsedValue = Number.parseInt(process.env.ES_CONCURRENCY, 10);

    if (Number.isInteger(parsedValue) && parsedValue > 0) {
      return parsedValue;
    }

    throw new TypeError('The ES_CONCURRENCY environment variable must be an integer greater than 0');
  }

  return 10;
};
```